mass_mailing_partner/views/res_partner_view.xml

```diff
@@ -35,5 +35,14 @@
     </field>
 </record>
 
+<act_window name="Add to mailing list"
+    res_model="partner.mail.list.wizard"
+    src_model="res.partner"
+    view_mode="form"
+    multi="True"
```

Comment:
If you remove multi, you will allow this action to be triggered from the contact form.